uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/components/annual-entitlements.component.js

```diff
@@ -134,8 +141,8 @@
       }
 
       /**
-       * Opens the Annual entitlement change log modal for the current
-       * contact and the given period.
+       * Opens the Annual entitlement change log modal
+       * for the current contact and the given period
```

Comment:
"for the contact, absence period, and absence type provided"

It would also be nice to mention that it configures the return URL to the current one.